.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: java
+before_cache:
```

Comment:
The docs linked in the commit message explain how this works, I'll add a comment inline too.

Basically, travis configures java/gradle builds automatically if the repo contains a `build.gradle` (see https://docs.travis-ci.com/user/languages/java/#Projects-Using-Gradle)